spec/addressable/template_spec.rb

```diff
@@ -1293,6 +1323,12 @@ def self.match(name)
     end
   end
 
+  describe 'inspect' do
+    it "returns a string representation of the MatchData's state" do
+      expect(its.inspect).to match(/\A#<Addressable::Template::MatchData:0x[0-9a-f]{0,12} RESULT:{"foo"=>"ab", "bar"=>"cd"}>\z/)
```

Comment:
This is related to the rest of the pull request insofar as Coveralls was failing because I removed a few lines of covered code, meaning the overall percentage of covered code went down. This covers some uncovered code, which making Coveralls happy again. I’ll remove it or move it into another pull request if you don’t mind merging this pull requests with Coveralls failing.